include/sc2api/sc2_unit.h

```diff
@@ -214,6 +215,17 @@ class UnitPool {
     void ClearExisting();
     bool UnitExists(Tag tag);
 
+    const std::vector<const Unit*>& GetNewUnits() const noexcept { return units_newly_created_; };
```

Comment:
There is a nice shortcut for the returned values here:
`17:include/sc2api/sc2_unit.h:202:typedef std::vector<const Unit*> Units`

Could make the things shorter.